packages/react/src/themes/teams/components/Segment/segmentVariables.ts

```diff
@@ -11,19 +11,32 @@ export type SegmentColorSchemeMapping = TeamsSchemeMappingWithAreas<
 export interface SegmentVariables {
   colorScheme: SegmentColorSchemeMapping
   color: string
+  colorDisabled: string
   backgroundColor: string
-  padding: string
+  backgroundColorDisabled: string
+  borderColorDisabled: string
   borderRadius: string | number
-  boxShadowColor: string
+  borderStyle: string
+  borderWidth: string
+  boxShadow: string
+  padding: string
 }
 
 export default (siteVariables): SegmentVariables => {
   return {
     colorScheme: pickValuesFromColorScheme(siteVariables.colorScheme, segmentColorAreas),
     color: siteVariables.bodyColor,
+    colorDisabled: siteVariables.colors.grey[250],
+
     backgroundColor: siteVariables.bodyBackground,
-    padding: '1em',
+    backgroundColorDisabled: siteVariables.colors.grey[150],
+
+    borderColorDisabled: 'transparent',
     borderRadius: 0,
-    boxShadowColor: 'rgba(34,36,38,.15)',
+    borderStyle: 'solid',
+    borderWidth: '2px 0 0 0',
+
+    boxShadow: '0 1px 1px 1px rgba(34,36,38,.15)',
```

Comment:
50/50: can't say that replacement of `boxShadowColor` is something bad, but not sure that single variables is enough, for example we have 4 variables for border

https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow#Values

Reply:
but for border we have CSS properties we can set; there is no CSS `boxShadowColor` so keeping the `boxShadowColor` variable is difficult to achieve when we expose `boxShadow`; reason:
`boxShadowColor` variable needs to be part of `boxShadow` variable value in order to take effect and it's difficult to predict where it will land because of the syntax: https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow#Syntax

Reply:
Let's leave it for now, but I definitely want to find something better